Dockerfile

```diff
@@ -21,6 +21,22 @@ COPY deploy/config/local/pool.conf /usr/local/etc/php-fpm.d/pool.conf
 # www-data
 USER 82
 
+
+## target: ssh
+FROM base AS ssh
+
+RUN apk add --no-cache openssh bash
+
+RUN ssh-keygen -A 
+RUN adduser -h /home/ssh-user -s /bin/bash -D ssh-user
+RUN echo 'PasswordAuthentication yes' >> /etc/ssh/sshd_config
+RUN echo -n "ssh-user:test" | chpasswd
```

Comment:
Just to air on the side of caution, our cyber-security team scan our repos. Can we include the password in the .env file and reference it here with something like the following?

.env
```
MIGRATE_SSH_PASSWORD=test
```

Dockerfile
```
ARG MIGRATE_SSH_PASSWORD

...
RUN echo -n "ssh-user:${MIGRATE_SSH_PASSWORD}" | chpasswd
...
```

Makefile
```
migrate:
	docker compose up --profile ssh
```

Reply:
I'm not sure if that will work due to the funny way Compose and Dockerfile manage environments, but worth a go. The service should run so try:

`docker compose exec -it ssh -- bash`

Reply:
... and maybe check that `chpasswd` did what you wanted 👍🏽

Reply:
Thanks @wilson1000-MoJ ! I didn't know the security team would scan the repos. That's good to know.

I've made the changes, except I used `LOCAL_SSH_PASSWORD` (before I saw your comment). Is that ok, as I feel `LOCAL` is a good indication to only use it in a local environment.

Happy to rename to `MIGRATE_SSH_PASSWORD` if you prefer.

Reply:
`LOCAL_SSH_PASSWORD` Sounds good to me :)

Reply:
Nice, it's done & working nicely here, do you mind doing a final review?